tests/installation/bootloader_s390.pm

```diff
@@ -95,6 +95,12 @@ sub prepare_parmfile {
                   shorten_url($host . '/' . get_required_var('REPO_0') . "/LiveOS/squashfs.img") :
                   $host . '/' . get_var('REPO_999'));
             $params .= $root_line;
+
+            # add mandatory boot params
+            $params .= ' cio_ignore=all,!condev,!0.0.0150';
+            $params .= ' rd.zdev=dasd,0.0.0150' if (get_var('FORMAT_DASD') ne 'never');
```

Comment:
"FORMAT_DASD" : "pre_install",
is default setting in legacy code main.pm we don't make any use of this setting because we manipulate the disk via Agama, due to potential race condition. If we want to opt in or out of having DASD we need something more readable, perhaps we need new parameter `AGAMA_ACTIVATE_DASD=1`, so you can have the default from parmfile for instance:
` $params .= ' rd.zdev=dasd,0.0.0150' unless  get_var('AGAMA_ACTIVATE_DASD') ;`

Reply:
If we add a new parameter it could be more readable, yes, on my way